docs/modules/ROOT/pages/how-to/install-redis.adoc

```diff
@@ -0,0 +1,3 @@
+= Install Redis on {product}
+
+Do you want a managed Redis that includes Backups, Monitoring, Updates, and optionally a 99.99% SLA and 24x7 Support? Check out the https://products.vshn.ch/appcat/redis.html[MariaDB by VSHN product description^] and https://docs.appcat.ch/vshn-managed/redis/create.html[technical documentation including how to instantiate^].
```

Comment:
Do we really sell [four nines](https://uptime.is/four-nines)???

Reply:
done